syntax/rest.vim

```diff
@@ -77,7 +77,7 @@ highlight link restHost Label
 syntax match restKeyword '\c\v^\s*(GET|POST|PUT|DELETE|HEAD|PATCH|OPTIONS|TRACE)\s'
 highlight link restKeyword Macro
 
-syntax match restComment '\v^\s*(#|//).*$'
+syntax match restComment '\v\s*(#|//).*$'
```

Comment:
VRC doesn't support inline comments; each comment must be on its own line. This is to simplify the parsing process such as

```
https://localhost:9200
POST /resource/10
{
  "name": "John",
  "comment": "My # is 123345"
}
```
So I think this change will also high light the `# is 12345` part.